lib/rules/no-return-await.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @fileoverview Disallows unnecessary `return await`
+ * @author Jordan Harband
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+    meta: {
+        docs: {
+            description: "disallow unnecessary `return await`",
+            category: "Best Practices",
+            recommended: false // TODO: set to true
+        },
+        fixable: false,
+        schema: [
+        ]
+    },
+
+    create(context) {
+        return {
+            ReturnStatement(node) {
+                if (node.argument && node.argument.type === "AwaitExpression") {
+                    context.report({
+                        node,
```

Comment:
Nit: Maybe it would be better to report `node.argument` instead of `node`, (or even the `await` keyword with `sourceCode.getFirstToken(node.argument)` to make it clear that `await` is the problem.